vnext/JSI.Desktop.UnitTests/JsiRuntimeUnitTests.h

```diff
@@ -0,0 +1,49 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <jsi/jsi.h>
+
+#include <gtest/gtest.h>
+
+#include <functional>
+#include <memory>
+
+namespace facebook {
```

Comment:
I suggest using `Microsoft` as a root namespace.
`facebook::react`, et al, were chosen initially when this code base was intended to become part of Facebook's react-native code base.

This is no longer the case.

Note, there has been offline discussion about moving away from `facebook::`, but work is deferred for existing APIs for compatibility purposes.
New APIs can safely use the `Microsoft` namespace instead.

Reply:
This file actually comes from Facebook upstream. which is why the facebook::react namespace is used here.

---
In reply to: [298776076](https://github.com/microsoft/react-native-windows/pull/2677#discussion_r298776076) [](ancestors = 298776076)